confignode/src/main/java/org/apache/iotdb/confignode/manager/load/balancer/RouteBalancer.java

```diff
@@ -46,8 +48,13 @@ public Map<TConsensusGroupId, TRegionReplicaSet> genRealTimeRoutingPolicy(
   }
 
   private IRouter genRouter() {
-    // TODO: The Router should be configurable
-    return new LoadScoreGreedyRouter(getLoadManager().getAllLoadScores());
+    String policy = ConfigNodeDescriptor.getInstance().getConf().getRoutingPolicy();
+    if (policy.equals("leader")) {
```

Comment:
extract “leader” to final static ....    other place could use together.

Reply:
Fixed~